app/models/admin_user.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+# AdminUser represents an administrative user with special privileges.
+class AdminUser < ApplicationRecord
```

Comment:
Why are we using two models for user?

Reply:
changed apu, i am using users model only to handle admin users and normal users